### ISSUE TITLE
MLIL display crash in x64 with base address above 0x100000000

### ISSUE THREAD

**Issue by SensorLock** [Vector35/binaryninja-api#2408]
**Describe the bug**
Crash due to access violation when displaying MLIL of a x64 Windows driver with a base address above 0x100000000.

**To Reproduce**
1. Open Binary Ninja
2. Open C:\Windows\System32\drivers\dxgmms1.sys (current Windows 10; driver version 10.0.19041.906)
3. Set UI view to LLIL
4. Go to address 0x1c005ad35
5. Set UI to MLIL
6. Experience a binary ninja crash due to access violation on deref of 0x18

If you stop before step 5 and try to list out the instructions from python you can see:
```
>>> list(current_mlil.instructions)
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "C:\...\AppData\Local\Vector35\BinaryNinja\plugins\..\python\binaryninja\mediumlevelil.py", line 368, in __repr__
    return "<il: %s>" % str(self)
  File "C:\...\AppData\Local\Vector35\BinaryNinja\plugins\..\python\binaryninja\mediumlevelil.py", line 359, in __str__
    tokens = self.tokens
  File "C:\...\AppData\Local\Vector35\BinaryNinja\plugins\..\python\binaryninja\mediumlevelil.py", line 405, in tokens
    if not core.BNGetMediumLevelILInstructionText(self._function.handle, self._function.source_function.handle,
OSError: exception: access violation reading 0x0000000000000018
```

Digging into the MLIL instruction list, the first item that throws exception during repr appears to have a 64-bit pointer truncated to 32-bits in the value:
```
hex(current_mlil[20].operands[1].operands[1].operands[0])
0xc002e0a0
```

That value should be 0x1c002e0a0, the import address for g_IsInternalRelease.

**Expected behavior**
Don't crash, raise error.
Expected view of MLIL in UI to render.
Expected the constant to have 1 in the upper 32 bits.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 10

**Additional context**
Observed in 2.3.2779-dev, introduced at some point after 2.2.2486-dev.

**Comment by psifertex** [comment]
We've released an interim patch that will at least fix the critical crashing issue in builds >= 2800 (should be live on dev in a few minutes) but I'm leaving this bug open because there's still a different underlying issue that we are still investigating.

**Comment by plafosse** [comment]
Fixed in > 2.3.2800
